src/config.c

```diff
@@ -272,6 +778,14 @@ int ReadConfig(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
                     "notice",
                     "Setting default series ENCODING to: %s",
                     ChunkTypeToString(TSGlobalConfig.options));
+
+    if (isDeprecated && showDeprecationWarning) {
```

Comment:
should use `LOG_DEPRECATED_OPTION` here

Reply:
How exactly? There is a completely different message used here compared to what the `LOG_DEPRECATED_OPTION` does, which was created only to show a warning for one option which has been renamed to a new name.

Reply:
Sorry, I wasn't clear here at all. If `isDeprecated && showDeprecationWarning`, does that not mean that you already showed the `LOG_DEPRECATED_OPTION` somewhere previously in the function? The user already got a warning that they're using a deprecated configuration option. Do they need to have another warning?

Reply:
Yes, the context is slightly different here.

1. The `LOG_DEPRECATED_OPTION` is used to provide a warning message that an option has a new name.
2. The `RedisModule_Log` line is used here to provide a warning message about using the deprecated load-time configuration options.

So those are two different kinds of warnings. One is about the renaming, and the other is about using the load-time module arguments.